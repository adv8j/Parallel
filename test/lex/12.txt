@Task
{
    int ct = 5;
    for i in 1..ct;
}
