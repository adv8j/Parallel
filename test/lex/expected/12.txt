TASK
LBRACE
INT
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
FOR
IDENTIFIER
IN
INT_LITERAL
RANGE
IDENTIFIER
SEMICOLON
RBRACE
